layouts/documents.html

```diff
@@ -0,0 +1,2 @@
+<h1 id="documents">Documents</h1>
```

Comment:
needs to be changed to Other

Reply:
As per my earlier comment, should stay as Documents

Reply:
We discussed internally and decided that this should be categorized under 'Other

Reply:
https://github.com/cerner/fhir.cerner.com/commit/876a7b1c830bb7a162e25dde5f9fc3a2bfc8dd02